brave/browser-laptop
# Adding the ability to toggle site shield settings via the context menu.

Resolves #7574

## Submitter Checklist:


## Test Plan:
 1. Visit a web page (Does not apply to internal about: pages)
 2. Note the current shield status (Enabled/Disabled) via the Brave Icon in the Navigation Bar.
 3. Right click anywhere in the page to trigger the context menu.
 4. Ensure that the item text implies the action opposite to the current shields status, (If shields are enabled, the item should say Disable Shields, else, Enable Shields)
 5. Select the menu item, ensure the shield status updates appropriately and that the page refreshes. 
 6. Ensure that the shields are functioning as indicated in the UI
 7. Also confirm that the shield status can be toggled via the bravery panel as expected 

## Reviewer Checklist:


Tests



cc: @jonathansampson
